packages/plugin-compat/extra/typescript/gen-typescript-patch.js

```diff
@@ -85,12 +85,19 @@ const SLICES = [
     onto: `7893f689fedcfe28a499d119d34844840bce9160`,
     range: `>=4.5.2 <4.6`,
   },
-  // https://github.com/merceyz/TypeScript/tree/merceyz/pnp-4.6
+  // https://github.com/merceyz/TypeScript/tree/merceyz/pnp-4.6-beta
   {
     from: `b8468c021393b2c36d6828ff2d3fc8665e09fd73`,
     to: `b8468c021393b2c36d6828ff2d3fc8665e09fd73`,
     onto: `40c4fac04935b84f16e8fed8ce730a86b330ba12`,
-    range: `>=4.6.0-beta`,
+    range: `4.6.0-beta`,
+  },
```

Comment:
It doesn't cost much to keep it due to the compression but I'll remove it since people using the TypeScript beta are likely to upgrade to the RC quickly. Correct, it was only out in canary releases.